lib/blogs_api_web/auth/guardian.ex

```diff
@@ -14,4 +17,24 @@ defmodule BlogsApiWeb.Auth.Guardian do
     |> Map.get("sub")
     |> BlogsApi.fetch_user()
   end
+
+  def authenticate(email, password) do
+    with {:ok, user} <- Login.get_by_email(email) do
+      case validate_password(password, user.encrypted_password) do
+        true ->
+          create_token(user)
+        false ->
+          {:error, :bad_request}
+      end
+    end
+  end
```

Comment:
Neste caso, também acredito que a requisição está correta, apenas a senha está incorreta, então poderia ser um status code `401 - unauthorized`, que tal?

> [401 Unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401)
Although the HTTP standard specifies "unauthorized", semantically this response means "unauthenticated". That is, the client must authenticate itself to get the requested response.

[Referência](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses)

```suggestion
  def authenticate(email, password) do
    with {:ok, user} <- Login.get_by_email(email) do
      case validate_password(password, user.encrypted_password) do
        true ->
          create_token(user)
        false ->
          {:error, :unauthorized}
      end
    end
  end
```